brave-kafka/pom.xml

```diff
@@ -56,6 +56,10 @@
       <groupId>io.zipkin.brave</groupId>
       <artifactId>brave</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.zipkin.java</groupId>
```

Comment:
Without this dependency compilation fails with:
```
INFO] Compiling 11 source files to /Users/llinder/git/smartoss/smartthings-brave/brave-kafka/target/classes
/Users/llinder/git/smartoss/smartthings-brave/brave-kafka/src/main/java/smartthings/brave/kafka/producers/BaseTracingProducerInterceptor.java:53: error: cannot access Endpoint
      .remoteEndpoint(kafkaEndpoint);
      ^
  class file for zipkin.Endpoint not found
```

From what I can tell this shouldn't be happening but it doesn't seem like the compiler can differentiate the overloaded method `remoteEndpoint`, one of which takes the messing class.

Reply:
yep. was a mistake to overload like this in hindsight. However, you can mark this dep optional, as it is only compile-required

Reply:
Cool thanks for pointing that out.  I'll update this to make it optional.